testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -437,6 +439,21 @@ fn simple_neon_integration() {
             .unwrap()
             .stacks_block_height;
 
+    // query for prometheus metrics
+    #[cfg(feature = "monitoring_prom")]
```

Comment:
For testing purposes, should we just always enable this flag?

Reply:
You mean in the testing CI, or via the Cargo.toml?

The last time I looked into this (which admittedly, was over a year ago, and Cargo development can be comparatively speedy), you couldn't set default features for testing (as opposed to `dev` profile). So, my understanding is that we'd have to make `monitoring_prom` a default feature in `dev` if we wanted it enabled in tests.

Reply:
> So, my understanding is that we'd have to make monitoring_prom a default feature in dev if we wanted it enabled in tests.

I think that's probably fine.  Could an alternative here be to use `#[cfg(any(feature = "monitoring_prom", feature = "testing"))]`?

Reply:
> Could an alternative here be to use #[cfg(any(feature = "monitoring_prom", feature = "testing"))]?

I think that would need to be placed in the `stackslib` dependency, but it would still be a problem getting it to default "on" when run as a test: the `testing` feature would only default as on for the `stackslib` dependency, not the `stacks-node` binary.

Realistically, it looks like the only way to really get this to work without changing the feature flag set during testing is to add `monitoring_prom` to the `defaults`.

Reply:
Okay, is there any way at all we can test this in CI that doesn't also require `monitoring_prom` to be set by default?

Reply:
Yes, definitely, we just need to add the feature flag to this command: 

https://github.com/stacks-network/actions/blob/129b799256eb97eef7fc15f5679c1295bec4279a/stacks-core/cache/build-cache/action.yml#L71

Reply:
Okay, can you do that as part of this PR?

Reply:
Sure -- I opened a PR for it. https://github.com/stacks-network/actions/pull/6

Reply:
Great, thanks!